scripts/features/crystal.sh

```diff
@@ -10,8 +10,8 @@ touch /home/vagrant/.homestead-features/crystal
 chown -Rf vagrant:vagrant /home/vagrant/.homestead-features
 
 # Install Crystal Programming Language Support
-apt-key adv --keyserver hkp://keys.gnupg.net:80 --recv-keys 09617FD37CC06B54
-echo "deb https://dist.crystal-lang.org/apt crystal main" | tee /etc/apt/sources.list.d/crystal.list
+curl -sL "https://keybase.io/crystal/pgp_keys.asc" | sudo apt-key add -
+echo "deb https://dist.crystal-lang.org/apt crystal main" | sudo tee /etc/apt/sources.list.d/crystal.list
```

Comment:
`sudo` isn't required here because this script is run as root.

Reply:
OK. Got it. Do you want me to fix it or you want to keep it? @svpernova09